src/ApiClient.php

```diff
@@ -52,35 +57,74 @@ public function search(string $searchTerm): array
     }
 
     /**
-     * Get historical data for a symbol.
+     * Get historical data for a symbol (depreciated).
+     *
+     * DEPRECIATION NOTICE:
+     * In future versions, this function will be removed.
+     * Please consider using getHistoricalQuoteData() instead.
```

Comment:
Would you please replace with block with `@deprecated In future versions, this function will be removed. Please consider using getHistoricalQuoteData() instead.` (in a single line). That will allow IDEs to detect the deprecation message and highlight it in the code.

Will do a proper review later today.

Reply:
<img width="631" alt="Screen Shot 2021-02-26 at 9 33 05 AM" src="https://user-images.githubusercontent.com/16628119/109320571-c33af780-7815-11eb-84e7-32f6d6f6a9f0.png">

Oh wow, fancy. I didn't even know that was a feature.